client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/toggle-activate-monitoring/index.tsx

```diff
@@ -153,19 +162,44 @@ export default function ToggleActivateMonitoring( {
 		);
 	}
 
+	const displayablePopover = shouldDisplayUpgradePopover ? (
+		<UpgradePopover
+			context={ statusContentRef.current }
+			isVisible={ showTooltip }
+			position="bottom left"
+		/>
+	) : (
+		<>
+			{ tooltip && (
+				<Tooltip
+					id={ tooltipId }
+					context={ statusContentRef.current }
+					isVisible={ showTooltip }
+					position="bottom"
+					className="sites-overview__tooltip"
+				>
+					{ tooltip }
+				</Tooltip>
+			) }
+		</>
+	);
+
 	return (
 		<>
 			<span
+				className="toggle-activate-monitoring__toggle-button"
+				onMouseDown={ handleHideTooltip }
+				onMouseEnter={ handleShowTooltip }
+				onMouseLeave={ handleHideTooltip }
 				ref={ statusContentRef }
 				role="button"
 				tabIndex={ 0 }
-				onMouseEnter={ handleShowTooltip }
-				onMouseLeave={ handleHideTooltip }
-				onMouseDown={ handleHideTooltip }
-				className="toggle-activate-monitoring__toggle-button"
 			>
 				{ toggleContent }
+
+				{ displayablePopover }
```

Comment:
This needs to be part of the toggle button so that it retains the hover state when we move the cursor over the popover.